### PR TITLE
correct usage of the CrateDB name

### PR DESCRIPTION
**Pull request by nslater**

### CODE REVIEW

**Comment by nslater** [comment]
@m-kharbat can I have an ack based ONLY on commit b6c3481 please?

the other commits listed on this pr are commits from the nomi/fix-doc-wrapping branch. I dont know why this is showing up, as I have rebased against that branch and the base on this pr is that branch, so they should be invisible. but regardless, once that branch is merged to master, I will rebase this branch against master, and will only apply b6c3481

**Comment by nslater** [comment]
@m-kharbat okay rebased so ignore my previous comment :)

this is good to be reviewed *as is*

File: blackbox/docs/hello.txt [hunk 1]
```diff
@@ -25,7 +25,7 @@ First let's connect to a running node::
     +------------------------+-----------+---------+-----------+---------+
     | server_url             | node_name | version | connected | message |
     +------------------------+-----------+---------+-----------+---------+
-    | http://127.0.0.1:...   | crate     | ...     | TRUE      | OK      |
+    | http://127.0.0.1:...   | my_node   | ...     | TRUE      | OK      |
```
Comment (m-kharbat) [inline blackbox/docs/hello.txt]: shouldn't this test fail ?

File: blackbox/docs/sql/scalar.txt [hunk 2]
```diff
@@ -233,11 +233,11 @@ Computes the SHA1 checksum of the given string.
 
 ::
 
-    cr> select sha1('crate');
+    cr> select sha1('foo');
     +------------------------------------------+
-    | sha1('crate')                            |
+    | sha1('foo')                            |
```
Comment (m-kharbat) [inline blackbox/docs/sql/scalar.txt]: table formatting, this test should also fail

File: blackbox/docs/sql/system.txt [hunk 3]
```diff
@@ -1036,7 +1036,7 @@ system column in the query::
     +---------------+--------...-+-----...-+------------+
     | _node['name'] | job_id     | name    | used_bytes |
     +---------------+--------...-+-----...-+------------+
-    | crate         | ...        | ...     | ...        |
+    | my_node       | ...        | ...     | ...        |
```
Comment (m-kharbat) [inline blackbox/docs/sql/system.txt]: same comment

**Pull request merged**